docs/source/openshift/openshift-client-binary.adoc

```diff
@@ -109,36 +108,29 @@ $ minishift logs
 ----
 
 [[update-openshift-config]]
-== Updating OpenShift configuration
+== Updating OpenShift Configuration
 
-While OpenShift is running, you can view and change the master or the
-node configuration of your cluster.
+While OpenShift is running, you can view and change the master or the node configuration of your cluster.
 
-To view the current OpenShift master configuration
-(_master-config.yaml_), run the following command:
+To view the OpenShift master configuration file *_master-config.yaml_*, run the following command:
 
 ----
 $ minishift openshift config view
 ----
 
-To show the node configuration instead of the master configuration,
-specify the `target` flag.
+To show the node configuration instead of the master configuration, specify the `target` flag.
 
-For details about the `view` command, see the link:../command-ref/minishift_openshift_config_view{outfilesuffix}[synopsis]
-command reference.
+For details about the `view` sub-command, see the xref:../command-ref/minishift_openshift_config_view.adoc#minishift-openshift-config-view[minishift openshift config view] synopsis.
 
 NOTE: After you update the OpenShift configuration, OpenShift will transparently restart.
 
 [[example-config-cors]]
-=== Example: Configuring cross-origin resource sharing
+.Example: Configuring cross-origin resource sharing
```

Comment:
This makes the entry disappear from the TOC and makes this example hard to find. I think it is better to keep these as sections. It gives us entries in the TOC and I also feel it is a bit too much content for the '.Example' form.